src/app/container/container.component.html

```diff
@@ -261,6 +261,14 @@ <h3>Source Repositories</h3>
         </div>
       </div>
     </div>
+    <div class="panel panel-default mb-3" *ngIf="isLoggedIn">
+      <div class="panel-heading">
+        <h3>Collections</h3>
```

Comment:
I know you already merged, but...

This UI implies to me it would have a list of collections that the tool belongs to, but it only ever has a button to add to a collection.

I suggest you either show the list of already existing collections the tool belongs to or tweak the UI  or somehow make it clearer UI-wise that there will not be a list here -- make it look different than other panels somehow?

Reply:
It's going to be added later (the collections it's in)